calico_kubernetes/logutils.py

```diff
@@ -3,7 +3,7 @@
 import os
 import sys
 
-LOG_DIR = '/var/log/calico/kubernetes'
+LOG_DIR = '/var/log/calico/kubernetes/exec-plugin/'
```

Comment:
I think that was assuming that we'd use svlogd. I'd prefer keeping it as-is for now; if we add a policy agent then we can refactor the logs then.